include/base_interface.h

```diff
@@ -106,11 +110,24 @@ class BaseInterface : public ParameterAcceptor, public SimulatorAccess<dim,space
    */
   void init ();
 
-
   /**
-     * Postprocess the newly generated triangulation.
-     */
-  virtual void postprocess_newly_created_triangulation(Triangulation<dim, spacedim> &tria) const;
+   * Override this function in your derived interface in order to
+   * connect to the signals, which are defined in the struct Signals.
+   *
+   * Example of implementation:
+   * @code
+   * auto &signals = this->get_signals();
+   * signals.fix_initial_conditions.connect(
+   *         [this](VEC &y, VEC &y_dot)
+   *          {
+   *            y=1;
+   *            y_dot=0;
+   *          });
+   * @endcode
+   *
+   * An example of implementation is given in the poisson_problem_signals.h file.
+   */
+  virtual void connect_to_signals() const {}
```

Comment:
Remove the `{}` from here, and put the default implementation in the cc file, or further down.